db/structure.sql

```diff
@@ -130,7 +123,7 @@ CREATE TYPE user_status_enum AS ENUM (
 
 CREATE FUNCTION maptile_for_point(bigint, bigint, integer) RETURNS integer
     LANGUAGE c STRICT
-    AS '$libdir/libpgosm', 'maptile_for_point';
+    AS '/srv/openstreetmap-website/db/functions/libpgosm.so', 'maptile_for_point';
```

Comment:
These changes likely do not belong here, as they are the result of setting up your local environment.
Make the changes include only relevant lines (adding `notes_count` and `diary_comments_count` in `users` table). Not sure why this file gets so vastly different - i noticed this problem also while setting up my dev environment.

Reply:
Because it is sensitive to things like where you have the 3GL functions installed. It can also vary in non-meaningful ways if you're running a different version of postgres as it's basically just the output of `pg_dump`.

Reply:
Then it would make things easier and to just `.gitignore` the whole file.... unless you use the diff for checking/verifying the effects of the changes on the actual DB.

Reply:
Oh I'd love not to have it, but people insisted on having a schema...